core/embed/trezorhal/stm32f4/usb/usbd_def.h

```diff
@@ -272,6 +272,8 @@ typedef struct _USBD_HandleTypeDef
 #define MAX(a, b)  (((a) > (b)) ? (a) : (b))
 #endif
 
+#define UNCONST(X) ((uint8_t *)(X))
```

Comment:
Not that it is new in this PR but this whole section seems misplaced, and we have MIN, MAX macros in common.h too. In future, we might wanna consider refactoring this too

Reply:
Yes, I agree with this, but I'm not sure if we should include `common.h` here, especially since we are gradually moving towards eliminating it altogether. I would prefer to wait with this until the fate of `common.h` is determined.

Reply:
Fine with me. I created an issue for that https://github.com/trezor/trezor-firmware/issues/3948